core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -781,6 +800,17 @@ private void executeSelect(
         currentCursor = factory.getCursor(sqlExecutionContext);
         prepareRowDescription();
         sendCursor();
+        send();
+    }
+
+    @Nullable
+    private CharSequence getStatementName(long lo, long hi) {
+        CharacterStoreEntry e = portalCharacterStore.newEntry();
+        CharSequence statementName = null;
+        if (hi - lo > 0 && Chars.utf8Decode(lo, hi, e)) {
```

Comment:
`utf8Decode` returns decoding status. It cannot be ignored as it indicates corrupt message. Usually when uff8 cannot be decoded BadProtocolException is being thrown